src/components/AddPeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -15,24 +15,10 @@
         <v-icon color="#38598A">mdi-account-multiple-plus</v-icon>
         <label class="people-roles-title"><strong>People and Roles</strong></label>
       </div>
-
-      <!-- Summary Warning -->
-      <div v-if="showErrorSummary" class="people-roles-invalid-message">
-        <span>
-          <v-icon color="blue darken-2">mdi-information-outline</v-icon>
-          This step is not complete.
-          <router-link
-            id="router-link"
-            :to="{ path: `/${RouteNames.ADD_PEOPLE_AND_ROLES}`, query: { showErrors: true } }"
-          >
-            Return to this step to complete it.
-          </router-link>
-        </span>
-      </div>
     </div>
 
     <!-- List Display Section -->
-    <div id="people-roles-list">
+    <div id="people-roles-list" :class="{ 'invalid-section': showErrorSummary && getValidateSteps }">
```

Comment:
I'll probably rename "getValidateSteps" at some point because it appears to return the validate steps, but it actually returns a boolean indicating whether the user has tried to submit a filing. Maybe I'll call the state variable "doValidate" or "validateState". Thoughts?

Reply:
Yea, was thinking on this as well! I'm fine with anything